optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/TestGenValueFactTest.java

```diff
@@ -20,14 +20,12 @@
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
-
+import static org.junit.Assert.*;
 import org.junit.Test;
 import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
 import org.optaplanner.core.impl.testdata.domain.TestdataValue;
 import org.optaplanner.core.impl.testdata.domain.collection.TestdataEntityCollectionPropertyEntity;
 
-import static org.junit.Assert.*;
-
```

Comment:
@yurloc: I understand that you are going to propose an alternative fix, so the PR is not required, so I won't fix this one.

I would import the rules if there were some for NetBeans, or they were defined in a non-IDE specific way.